src/common/SelectableOptionRow.js

```diff
@@ -33,8 +33,8 @@ type Props<TItemKey: string | number> = $ReadOnly<{|
   // position that requires it).
   itemKey: TItemKey,
 
-  subtitle: string,
```

Comment:
Could this commit message be `SelectableOptionRow [nfc]: Reorder variables.` instead of `SelectableOptionRow [nfc]: Tiny refactor.`? That makes it more clear what's going on.